### PR TITLE
Documentation for 1.4

### LINKED ISSUE #170: List of Must have Doc updates for upcoming  1.4
- [ ] Advanced policy with access to pluginContext
- [ ] API endpoint for service\api -endpoints, policies, schema etc. 
- [ ] EG behind proxy 
- [x] EG with redis Sentinels 
- [x] Multi-Auth
- [x] App name alias support
- [ ] Importing data and building test data set
   + [x] Update CLI example to show how to set keyId keySecret when creating  
- [x] Redis Migration 

[Update with whatever is missing]

### PR DESCRIPTION
**Pull request by XVincentX**
Connect #170 
Closes #170 
Connects ExpressGateway/express-gateway#533

### CODE REVIEW

File: docs/admin/policies/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+##### Response: 200
+
+```json
+[
+	"proxy",
+	"key-auth"
+]
+```
+
+### Enables a policiy
```
Comment (altsang) [inline docs/admin/policies/index.md]: `policy`

File: docs/configuration/system.config.yml/db.md [hunk 2]
```diff
@@ -9,12 +9,15 @@ This section defines how Express Gateway stores it data.
...
-    - Data is reset on each process restart
-    - Data is intact during Hot Reload operation
-    - No need for Redis instance
-    - Ideal for development
-    - Should not be used in production
+
+- Data is reset on each process restart
+- Data is intact during Hot Reload operation
+- No need for Redis instance
+- Ideal for development
```
Comment (altsang) [inline docs/configuration/system.config.yml/db.md]: `Ideal for development and demos`

**Review by altsang — CHANGES_REQUESTED** [review]
should mention that `app_name` has to be unique by user but that across an EG instance there may be duplicate `app_names` - bsst place to include this info?

File: docs/policies/proxy.md [hunk 3]
```diff
@@ -64,8 +64,11 @@ pipelines:
 
 #### Load balancing strategies
 
+* `static`: This strategy routes each client request to the fixed assigned `url`
```
Comment (DrMegavolt) [inline docs/policies/proxy.md]: this is just a technical detail how it is implemented. This is not a load balancing strategy.

File: docs/admin/policies/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+doc-order: 10.6
+---
+
+### Overview Policies API
+
+Express Gateway's Admin API supports querying, enabling and disabling available policies in the configuration.
+
+### Get enabled policies
+
+##### Request: `GET /policies`
```
Comment (DrMegavolt) [inline docs/admin/policies/index.md:13]: Is there are doc for pipelines/apiEndpoints/ServiceEndpoints ?

File: docs/policies/proxy.md (hunk 3 above)
Comment (XVincentX) [reply docs/policies/proxy.md]: Shall I remove it?

Comment (DrMegavolt) [reply docs/policies/proxy.md]: yes, I'd remove it

**Comment by XVincentX** [comment]
Ok, done!

File: docs/admin/policies/index.md (hunk 4 above)
Comment (XVincentX) [reply docs/admin/policies/index.md:13]: That is still missing, unfortunately.

**Pull request merged**